src/main/scala/nl.knaw.dans.easy.deposit/EasyDepositApiApp.scala

```diff
@@ -293,10 +293,13 @@ class EasyDepositApiApp(configuration: Configuration) extends DebugEnhancedLoggi
     else Success(())
   }
 
-  private def contentTypeAnythingButZip(contentType: Option[String]): Try[Unit] = {
-    if (contentType.exists(str => str.trim.nonEmpty && !str.trim.matches(contentTypeZipPattern)))
-      Success(())
-    else Failure(InvalidContentTypeException(contentType, "must not be a zip."))
+  private def contentTypeAnythingBut(contentType: Option[String]): Try[Unit] = {
+    if (contentType.map(_.trim.toLowerCase).exists(str =>
+      str.nonEmpty &&
+      !str.matches(contentTypeZipPattern) &&
```

Comment:
1. would it be wise to swap the `matches` and `startsWith` tests: the latter is faster and can more easily fail
2. formatting please 😉